django_netjsonconfig/widgets.py

```diff
@@ -27,8 +27,8 @@ def media(self):
     def render(self, name, value, attrs={}):
         attrs['class'] = 'vLargeTextField jsoneditor-raw'
         html = """
-<a href="#advanced_editor" class="button json-editor-btn-edit normal-mode">{0}</a>
```

Comment:
@nemesisdesign I just updated the PR and have removed the useless button and hooked the event properly to the new button instead. Waiting for your review.